pyrep/objects/object.py

```diff
@@ -93,18 +93,19 @@ def set_name(self, name: str) -> None:
         """
         sim.simSetObjectName(self._handle, name)
 
-    def get_position(self, relative_to=None) -> List[float]:
+    def get_position(self, relative_to=None) -> np.ndarray:
         """Gets the position of this object.
 
         :param relative_to: Indicates relative to which reference frame we want
             the position. Specify None to retrieve the absolute position, or an
             Object relative to whose reference frame we want the position.
-        :return: A list containing the x, y, z position of the object.
+        :return: An array containing the x, y, z position of the object.
         """
         relto = -1 if relative_to is None else relative_to.get_handle()
-        return sim.simGetObjectPosition(self._handle, relto)
+        position = sim.simGetObjectPosition(self._handle, relto)
+        return np.array(position, dtype=np.float64)
 
-    def set_position(self, position: List[float], relative_to=None,
+    def set_position(self, position, relative_to=None,
```

Comment:
Types: `position: Union[list, np.ndarray]`
(And for other occurrences in this pr)